cmd/fluxd/main.go

```diff
@@ -170,10 +170,10 @@ func main() {
 		k8sSecretDataKey         = fs.String("k8s-secret-data-key", "identity", "data key holding the private SSH key within the k8s secret")
 
 		// k8s-scope settings
-		k8sNamespaceWhitelist    = fs.StringSlice("k8s-namespace-whitelist", []string{}, "restrict the view of the cluster to the namespaces listed. All namespaces are included if this is not set")
-		k8sAllowNamespace        = fs.StringSlice("k8s-allow-namespace", []string{}, "restrict all operations to the provided namespaces")
-
-		k8sVerbosity             = fs.Int("k8s-verbosity", 0, "klog verbosity level")
+		k8sNamespaceWhitelist = fs.StringSlice("k8s-namespace-whitelist", []string{}, "restrict the view of the cluster to the namespaces listed. All namespaces are included if this is not set")
+		k8sAllowNamespace     = fs.StringSlice("k8s-allow-namespace", []string{}, "restrict all operations to the provided namespaces")
+		k8sVerbosity          = fs.Int("k8s-verbosity", 0, "klog verbosity level")
+		k8sDefaultNamespace   = fs.String("k8s-default-namespace", "default", "the namespace to use for resources where a namespace is not specified")
```

Comment:
`k8sVerbosity` is not a scope setting, can you move it to a separate block, like it was before?

Reply:
Also, I wouldn't set a default value for `k8s-default-namespace`. Using `default` would require setting it to empty when the user wants to supply a default namespace through a kubeconfig file.